src/app/shared/common/translate-with-slot/translate-with-slot.component.ts

```diff
@@ -22,6 +24,14 @@ export class TranslateWithSlotComponent implements OnChanges, OnDestroy {
   textAfter = '';
 
   constructor(private translationService: TranslateService) {
+    this.splitText();
+  }
+
+  ngOnInit() {
+    this.splitText();
+  }
```

Comment:
keep one of the two

Reply:
it didn't work with only one

Reply:
maybe replace translateService.get with translateService.stream

Reply:
looks like stream with only constructor works great, thanks!